voluncheer/opportunityboard/unittest_setup.py

```diff
@@ -0,0 +1,63 @@
+import datetime as dt
+
+from opportunityboard.models import Category
+from opportunityboard.models import Opportunity
+from opportunityboard.models import Subcategory
+from opportunityboard.models import Subsubcategory
+from profiles.models import Organization
+from profiles.models import User
+from profiles.models import UserType
+
+
+def setup_oppboard_tests(test_case):
```

Comment:
Good idea, moving this to a shared location. If I can make one suggestion though, a common pattern is to declare a new `TestCase` class that extends the `setUp` method. For example:

```
from django import test

class TestCase(test.TestCase):

    def setUp(self):
        super().setUp()
        # perform additional setup for this shared test case...
```

In the files that need this additional setup:

```
from opportunityboard.test import TestCase

class OpportunityTest(TestCase):

    # Optional additional setup...
    def setUp(self):
        super().setUp()

# Do tests...
```

WDYT?

Reply:
Awesome, thanks for the tip. I changed the tests accordingly.